test/utils/fieldUtils.js

```diff
@@ -118,6 +118,18 @@ class FieldDataTypes {
     return dateField;
   }
 
+  /**
+   * Enter the valid document file name to be uploaded
+   * @returns CCDStringField Object
+   */
+  async enterIntoDocumentField(value, id){
+    let css = await FIELDS.DOCUMENT.cssTag;
+    let docField = new CCDDocumentField(css, id);
+    await docField.uploadFile('C:\\test\\framework-structure.png');
```

Comment:
as discussed we would need to try and find a way to use a relative path for this to work in the pipeline

Reply:
Investigating...

Reply:
Done.